src/jit/gentree.cpp

```diff
@@ -620,6 +674,15 @@ void Compiler::fgWalkAllTreesPre(fgWalkPreFn* visitor, void* pCallBackData)
 //
 // Return Value:
 //     None
+//
+// Notes:
+//     This used to be done prior to calling TreeNodeInfoInit(). However, it is problematic because there are
+//     places that set the register in Lowering.
+//     Previously, this was worked around by querying the register value in TreeNodeInfo::Initialize()
+//     then using it to set the candidates on the node before clearing the register.
+//     That hack is now longer feasible as we move to eliminate TreeNodeInfo on GenTree nodes.
+//
+//     None
```

Comment:
I'm not sure what this note is telling me, since the code for gtClearReg seems pretty clear. It seems like this comment is perhaps misplaced, and should be in the caller someplace, explaining phase ordering there. (also, you now have two "None"? and "is now longer" => "is no longer")